library/LuaTools.cpp

```diff
@@ -554,6 +554,17 @@ static int dfhack_error(lua_State *L)
     return lua_error(L);
 }
 
+// replaces os.exit with a thrown exception
+static int dfhack_exit_error(lua_State *L)
```

Comment:
Was disabling `os.exit()` needed to debug a luacov issue? I'm totally fine with keeping this - nobody should be using it anyway.

Reply:
I should have explained that, sorry. Unmodified luacov can call os.exit() on configuration error. It won't cause df to exit, but it will bork the dfhack console. By default, this should never happen, but if users/developers create a .luacov file with errors, it can come up.

I also thought it was a good general protection against unruly scripts calling os.exit(), which, as you said, would never make sense for DFHack anyway.